ecs-service/variables.tf

```diff
@@ -7,6 +7,26 @@ variable "aws_region" {
   default = "us-east-1"
 }
 
+variable "dd_monitor_cpu_warning" {
+  type    = number
+  default = 80
```

Comment:
Takze default nechame takhle vysokej? Je to kvuli tomu, zes zmensil to okno na 5min avg?

Reply:
Nó, to by měl být jen alert abysme se dozvěděli o tom, že je jeden konkrétní task zaseknutej. Tady nejde o autoscaling a tím pádem ani o průměr na všech kontejnerech tasku.

Reply:
ajo... sorry rano :D

Reply:
V pohodě. 😄 Já si aspoň ujasním jestli jsem to nastavil dobře. 🙂